cliquet/resource/viewset.py

```diff
@@ -179,6 +179,11 @@ def get_record_schema(self, resource_cls, method):
                                              permissions=allowed_permissions,
                                              name='permissions')
         schema.add(permissions_node)
+
+        # XXX: If Cornice wouldn't recreate the schema on the fly.
```

Comment:
I can update the comment and link to those https://github.com/mozilla-services/cornice/labels/schema-validation

We should probably invest some time on this though :/